### PR TITLE
Add basic layout system documentation.

### PR DESCRIPTION
**Pull request by colinrtwhite**
Going to follow up with visual documentation for modifiers.

### CODE REVIEW

File: redwood-layout-schema/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Redwood Layout
+
+Redwood's layout system provides common layout primitives to support arranging and laying out widgets. It provides widgets for:
+
+- `Row`: Lays wigets out along the X axis (horizontally).
+- `Column`: Lays widgets out along the Y axis (vertically).
+- `Box`: Lays widgets out along the Z axis (on top of eachother).
+- `Spacer`: Adds space between widgets.
+
+Internally the layout system uses a common layout engine written in Kotlin Multiplatform to support consistent rendering across platforms. The system provides widget bindings for for Android Views (`redwood-layout-view`), iOS UiKit (`redwood-layout-uiview`), and Compose UI (`redwood-layout-composeui`).
```
Comment (JakeWharton) [inline redwood-layout-schema/README.md]: No DOM mention?

File: redwood-layout-schema/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+</p>
+
+### Spacer
+
+Adds space between widgets.
+
+```kotlin
+Column {
+    Text("Top")
+    Spacer(height = 24.dp)
```
Comment (JakeWharton) [inline redwood-layout-schema/README.md]: Indent seems to be 4 spaces throughout. Should be 2, but I would think the editorconfig set that correctly...

File: redwood-layout-schema/docs_images/box1.png [hunk 3]
```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```
Comment (JakeWharton) [inline redwood-layout-schema/docs_images/box1.png:1]: These are in LFS. Do we want to change our rules so only screenshot tests go into LFS? Or are we okay sending all pngs?

**Review by JakeWharton — APPROVED** [review]

File: redwood-layout-schema/README.md (hunk 2 above)
Comment (colinrtwhite) [reply redwood-layout-schema/README.md]: Ah yep - wrote this up in VS Code so the editorconfig didn't apply.

File: redwood-layout-schema/docs_images/box1.png (hunk 3 above)
Comment (colinrtwhite) [reply redwood-layout-schema/docs_images/box1.png:1]: Hmm I don't have a strong opinion. I imagine the docs are going to either be viewed on github.com, which correctly renders the images inline, or on the published docs site, which will host the images. Is there a downside to storing these with LFS?

Comment (JakeWharton) [reply redwood-layout-schema/docs_images/box1.png:1]: Mostly just philosophical. It breaks the concept of a self-contained repo. The screenshot test goldens can be refreshed by running the tests. These are more like part of the codebase.

Comment (colinrtwhite) [reply redwood-layout-schema/docs_images/box1.png:1]: @JakeWharton Good point, I'll clean up the code that generates these screenshots and add it to the repo.

Comment (colinrtwhite) [reply redwood-layout-schema/docs_images/box1.png:1]: Good point, I'll clean up the code that generates these screenshots and add it to the repo.

**Pull request merged**